scripts/cubemx-importer.stanza

```diff
@@ -477,9 +562,27 @@ public defn import-stm32-mcu (o:OutputStream, mcu:Stm32MCU):
   val gpio = gpio-pins(pins(mcu))
   do(println{os, _}, gpio)
 
+  val adc = adc-pins(pins(mcu))
+  do(println{os, _}, adc)
+
+  val dac = dac-pins(pins(mcu))
+  do(println{os, _}, dac)
+
   ; Next print the IP blocks we care about
   print-i2c(os, mcu)
+  print-can(os, mcu)
   print-spi(os, mcu)
   print-uart(os, mcu)
+  print-usart(os, mcu)
   print-i2s(os, mcu)
   print-oscillator(os, mcu)
+
+defn print-pins (pins:Tuple<Stm32Pin>):
```

Comment:
why not `defmethod print (o:OutputStream, pins:Tuple<Stm32Pin>)`?

Reply:
This is a simple print function for debugging. Not relevant to actual code